examples/example_project/src/lib.rs

```diff
@@ -0,0 +1,81 @@
+//! Simple fizzbuzz library
+//!
+//! Copied from https://github.com/matthewrudy/rust-fizzbuzz
+#![cfg_attr(test, feature(plugin))]
+#![cfg_attr(test, plugin(mutagen_plugin))]
+#![feature(custom_attribute)]
+
+#[cfg(test)]
+extern crate mutagen;
+
+/// This function adds two to its argument.
+///
+/// # Examples
+///
+/// ```
+/// use example_project::word_for;
+///
+/// assert_eq!("1",    word_for(1));
+/// assert_eq!("2",    word_for(2));
+/// assert_eq!("fizz", word_for(3));
+/// assert_eq!("4",    word_for(4));
+/// assert_eq!("buzz", word_for(5));
+///
+/// assert_eq!("fizz buzz", word_for(15));
+/// ```
+#[mutate]
+pub fn word_for(n: u32) -> String {
+    let mut string = String::new();
+
+    if n % 3 == 0 {
+        string.push_str("fizz ");
+    }
+    if n % 5 == 0 {
+        string.push_str("buzz ");
+    }
+
+    if string.len() > 0 {
+        string.pop();
+        return string;
+    }
+
+    return n.to_string();
+}
+
+#[cfg(test)]
+mod tests {
+
+    use super::word_for;
+    use super::mutagen;
+
+    #[test]
+    fn fizz_for_multiples_of_3() {
+        for _ in 0..100 {
+            mutagen::MU.next();
+
+            println!(
+                "Test {}: {} == {} => {}",
+                mutagen::MU.get(),
```

Comment:
The example should use the new methods here, no?

Reply:
Yes, it should.